### PR TITLE
Add build instructions to README

### PR DESCRIPTION
**Pull request by wan-nyan-wan**
I added Ubuntu and Mac build instructions to README.md.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,6 +29,29 @@ Road Map in 2020
 - Developer tools improvements: userland debugger, fuzzer, unit testing, etc.
 - GUI server
 
+Build Instructions
+------------------
+### Mac
```
Comment (nuta) [inline README.md:34]: How about `macOS` instead of `Mac`?

File: README.md [hunk 2]
```diff
@@ -29,6 +29,29 @@ Road Map in 2020
...
+```
+brew install llvm python qemu bochs i386-elf-grub xorris
```
Comment (nuta) [inline README.md:36]: nitpick: `xorris` -> `xorriso`

**Comment by nuta** [comment]
JFYI: Ignore a failure in CI. That's my fault :)

File: README.md (hunk 1 above)
Comment (wan-nyan-wan) [reply README.md]: I fixed it.

File: README.md (hunk 2 above)
Comment (wan-nyan-wan) [reply README.md]: I fixed it.

**Comment by wan-nyan-wan** [comment]
Thanks for codereview. I fixed them.

**Comment by nuta** [comment]
Thank you for your contribution :D

**Pull request merged**